ngraph/frontend/onnx_import/src/op/org.openvinotoolkit/experimental_detectron/detection_output.hpp

```diff
@@ -39,17 +39,17 @@ namespace ngraph
                     auto im_info = inputs[3];
 
                     DetectionOutput::Attributes attrs{};
-                    attrs.score_threshold = node.get_attribute_value<float>("score_threshold");
-                    attrs.nms_threshold = node.get_attribute_value<float>("nms_threshold");
-                    attrs.max_delta_log_wh = node.get_attribute_value<float>("max_delta_log_wh");
-                    attrs.num_classes = node.get_attribute_value<std::int64_t>("num_classes");
-                    attrs.post_nms_count = node.get_attribute_value<std::int64_t>("post_nms_count");
+                    attrs.score_threshold = node.get_attribute_value<float>("score_threshold", 0.05);
+                    attrs.nms_threshold = node.get_attribute_value<float>("nms_threshold", 0.5);
+                    attrs.max_delta_log_wh = node.get_attribute_value<float>("max_delta_log_wh", log(1000. / 16.));
+                    attrs.num_classes = node.get_attribute_value<std::int64_t>("num_classes", 81);
```

Comment:
num_classes default value is inherited from model_optimizer. Does anyone know what is the justification for this value?